packages/amplify-category-api/package.json

```diff
@@ -73,17 +73,17 @@
     "uuid": "^8.3.2"
   },
   "peerDependencies": {
-    "@aws-amplify/amplify-environment-parameters": "^1.4.0-beta.1",
-    "amplify-cli-core": "^4.0.0-beta.7",
-    "amplify-headless-interface": "^1.17.1-beta.2",
-    "amplify-prompts": "^2.6.4-beta.2",
-    "amplify-provider-awscloudformation": "^7.0.0-beta.6",
-    "amplify-util-headless-input": "^1.9.10-beta.2"
+    "@aws-amplify/amplify-environment-parameters": "^1.4.0-beta.2",
+    "amplify-cli-core": "^4.0.0-beta.8",
+    "amplify-headless-interface": "^1.17.1-beta.3",
+    "amplify-prompts": "^2.6.4-beta.3",
+    "@aws-amplify/amplify-provider-awscloudformation": "7.0.0-beta.8",
+    "amplify-util-headless-input": "1.9.10-beta.3"
```

Comment:
some of `^` removal are intentional to not clash with other release tags. All this will be reverted back after we GA.